docs/source/docker/getting_started/first_steps.rst

```diff
@@ -412,6 +412,36 @@ The corresponding Python command to submit a job would then be as follows:
     )
 
 
+Downloading
+-----------
+
+After a job has successfully run, a dataset with the selected samples
+and a tag with the name `initial-tag` are created. From there you can easily
+download the filenames for further processing.
+
+.. code-block:: python
+    :caption: Download the filenames for further processing
+    from lightly.api.api_workflow_client import ApiWorkflowClient
+
+    client = ApiWorkflowClient(token='123', dataset_id='xyz')
+    filenames = client.export_filenames_by_tag_name(
+        'initial-tag' # name of the datasets tag 
+    )
+    with open('filenames-of-initial-tag.txt', 'w') as f:
+        f.write(filenames)
+
+It is also possible to directly download the actual files themselves.
+.. code-block:: python
+    :caption: Directly download the files
+    from lightly.api.api_workflow_client import ApiWorkflowClient
+
+    client = ApiWorkflowClient(token='123', dataset_id='xyz')
+    client.download_dataset(
+        './my/output/path'  # path to where the files should be saved 
+        'initial-tag'       # name of the datasets tag
+    )
+
```

Comment:
this whole section doesn't format properly:
![image](https://user-images.githubusercontent.com/65946090/176118398-39c56a53-d0b5-405c-ac67-13d6254312b3.png)

Reply:
I guess adding a new line fixes the formatting? or how can I locally test this without rebuilding all experiments?